mllib/src/main/scala/org/apache/spark/ml/util/ReadWrite.scala

```diff
@@ -273,7 +273,29 @@ private[ml] object DefaultParamsReader {
       sparkVersion: String,
       params: JValue,
       metadata: JValue,
-      metadataJson: String)
+      metadataJson: String) {
+
+    /**
+     * Get the JSON value of the [[org.apache.spark.ml.param.Param]] of the given name.
+     * This can be useful for getting a Param value before an instance of [[Params]]
+     * is available.
+     */
+    def getParamValue(paramName: String): JValue = {
```

Comment:
This is a public API but it returns a type from a 3rd-party package. Shall we mark it private?